src/components/Card.js

```diff
@@ -1,29 +1,53 @@
 export default class Card {
-  constructor(data, cardSelector, handleImageClick) {
-    this._name = data.name;
-    this._link = data.link;
+  constructor(data, cardSelector, handleImageClick, handleDeleteClick,
+    handleLikeClick) {
+    this.name = data.name;
+    this.link = data.link; 
+    this._id = data._id; 
+    this.isLiked = data.isLiked;
```

Comment:
Needs correction: `isLiked` variable should be private. If you want to use it outside you should create a getter `getIsLiked`